examples/hello-world-script/setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+from setuptools_rust import RustExtension, Binding
+
+setup(
+    name="hello-world-script",
+    version="1.0",
+    rust_extensions=[
+        RustExtension(
+            {"hello-world-script": "hello_world.hello-world-script"},
+            binding=Binding.Exec,
+            script=True,
```

Comment:
To check I understand correctly, we won't need `examples/hello-world-script` when we drop `Binding.Exec` in 2.0?

Reply:
Yes.